app/components/Exchange/Exchange.jsx

```diff
@@ -37,6 +42,7 @@ import {Notification} from "bitshares-ui-style-guide";
 import PriceAlert from "./PriceAlert";
 import counterpart from "counterpart";
 import {updateGatewayBackers} from "common/gatewayUtils";
+import {Button} from "antd";
```

Comment:
Why did you import that button directly instaed of styleguide?

Reply:
I will fix it ASAP

Reply:
I will fix it asap

Reply:
I fixed it